TranslationMemoryManagementUtility/Sdl.Community.Utilities.TMTool.Tasks.RemapTMX/Sdl.Community.Utilities.TMTool.Tasks.RemapTMX.csproj

```diff
@@ -52,6 +52,10 @@
     <Reference Include="Sdl.LanguagePlatform.TranslationMemoryApi">
       <HintPath>$(MSBuildProgramFiles32)\SDL\SDL Trados Studio\Studio16\Sdl.LanguagePlatform.TranslationMemoryApi.dll</HintPath>
     </Reference>
+    <Reference Include="Sdl.Versioning, Version=17.0.0.0, Culture=neutral, PublicKeyToken=c28cdb26c445c888, processorArchitecture=MSIL">
```

Comment:
Clean up the reference here; should be &lt;Reference Include="Sdl.Versioning"&gt;...